From: Christoph Goehre <chris@sigxcpu.org>
Date: Tue, 17 Dec 2013 16:57:26 +0100
Subject: sogo-connector.xpt: prepare option for rebuild the *.xpt file

---
 components/Makefile | 19 +++++++++++++++++++
 1 file changed, 19 insertions(+)
 create mode 100644 components/Makefile

diff --git a/components/Makefile b/components/Makefile
new file mode 100644
index 0000000..b6f3037
--- /dev/null
+++ b/components/Makefile
@@ -0,0 +1,19 @@
+#!/usr/bin/make -f
+
+TYPELIB = /usr/lib/icedove-devel/sdk/bin/typelib.py
+XPIDL = /usr/lib/icedove-devel/sdk/bin/xpt.py
+ICEDOVEIDL = /usr/lib/icedove-devel/idl
+
+IDL = NotificationManager.xpt SyncProgressManager.xpt ContextManager.xpt
+
+sogo-connector.xpt: $(IDL)
+	python $(XPIDL) link $@ $^
+
+%.xpt: %.idl
+	python $(TYPELIB) -o $@ $< -I $(ICEDOVEIDL)
+
+clean:
+	rm -f sogo-connector.xpt $(IDL)
+
+.PHONY: clean
+
